pages/rules/rules.css

```diff
@@ -0,0 +1,61 @@
+.page-container {
+    height: 100vh;
+}
+
+/*Nav bar*/
+.navbar {
+    padding: 1rem 3rem;
+    border-bottom: 1px solid var(--tertiary-color);
```

Comment:
Primary and secondary is enough any further than that is confusing instead name the color with respective colorname.

Reply:
okay will make this change. Thanks for reviewing.